### PR TITLE
Merged SLE-12-SP1

### PR DESCRIPTION
**Pull request by mchf**

### CODE REVIEW

**Comment by mchf** [comment]
see also https://github.com/yast/yast-installation/pull/336

File: src/lib/network/network_autoyast.rb [hunk 1]
```diff
@@ -105,5 +190,17 @@ def merge_routing(instsys_routing, ay_routing)
...
+    def ay_networking_section
+      Yast.import "Profile"
+
+      ay_profile = Profile.current
+
+      return {} if ay_profile.nil? || ay_profile.empty?
+      return {} if ay_profile["networking"].nil?
+
+      ay_profile["networking"]
+    end
```
Comment (teclator) [inline src/lib/network/network_autoyast.rb]: NP: why not just

```
ay_profile = Profile.current || {}

ay_profile["networking"] || {}
```

Comment (mchf) [inline src/lib/network/network_autoyast.rb]: bcs I'm simple and i want to see explicitly what is return value in edge cases ... just say coding style.

File: package/yast2-network.changes [hunk 2]
```diff
@@ -1,4 +1,13 @@
 -------------------------------------------------------------------
+Mon Feb 22 11:52:09 UTC 2016 - mfilka@suse.com
+
+- bnc#960703
+  - sometimes wicked was started instead of NetworkManager in case
+    of AY installation. Moreover it could happen that both services
+    were "half-started" and network setup got broken.
+- 3.1.143
```
Comment (mvidner) [inline package/yast2-network.changes]: The changelog is missing the Infiniband change from commit 648260d00cb3e02cc100bba986c5824d1d75f9c3

File: src/include/network/routines.rb [hunk 3]
```diff
@@ -324,7 +324,9 @@ def ControllerType(hwdevice)
           return "atm"
         elsif subclass_id == 4
           return "isdn"
-        elsif subclass_id == 6
+        elsif subclass_id == 6 ## Should be PICMG?
+          return "ib"
+        elsif subclass_id == 7
```
Comment (mvidner) [inline src/include/network/routines.rb]: The actual code for Infiniband **is** merged, good.

**Comment by mvidner** [comment]
LGTM

**Comment by mchf** [comment]
Thanks for review

**Pull request merged**